include/simdjson/generic/numberparsing.h

```diff
@@ -293,7 +293,10 @@ simdjson_really_inline bool compute_float_64(int64_t power, uint64_t i, bool neg
 static bool parse_float_fallback(const uint8_t *ptr, double *outDouble) {
   *outDouble = simdjson::internal::from_chars((const char *)ptr);
   // We do not accept infinite values.
-  if (!std::isfinite(*outDouble)) {
+  if ((*outDouble > 1.7976931348623157E+308) || (*outDouble < -1.7976931348623157E+308)) {
```

Comment:
Would it work to use https://en.cppreference.com/w/cpp/types/numeric_limits/max instead? In my opinion, that would make the code self explaining.

Reply:
You are correct. Modified.

Reply:
@pauldreik Sadly, it appears to be unsupported under Visual Studio.